### PR TITLE
feat(oui-table): add modifier for action column

### PR DESCRIPTION
**Pull request by neolitec**
Signed-off-by: Kevin Manson kevin.manson@corp.ovh.com

### CODE REVIEW

File: packages/oui-table/README.md [hunk 1]
```diff
@@ -28,18 +29,27 @@ oui-table is a package which provides styles for the table component.
       <td data-title="Id" class="oui-table__cell">1</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 2]
```diff
@@ -28,18 +29,27 @@ oui-table is a package which provides styles for the table component.
...
+      </td>
     </tr>
     <tr class="oui-table__row" tabindex="0">
       <td data-title="Filename" class="oui-table__cell">abc</td>
       <td data-title="Id" class="oui-table__cell">2</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 3]
```diff
@@ -28,18 +29,27 @@ oui-table is a package which provides styles for the table component.
...
+      </td>
     </tr>
     <tr class="oui-table__row" tabindex="0">
       <td data-title="Filename" class="oui-table__cell">abc</td>
       <td data-title="Id" class="oui-table__cell">3</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 4]
```diff
@@ -65,6 +76,9 @@ oui-table is a package which provides styles for the table component.
       <td data-title="Id" class="oui-table__cell">1</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 5]
```diff
@@ -73,6 +87,9 @@ oui-table is a package which provides styles for the table component.
       <td data-title="Id" class="oui-table__cell">2</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 6]
```diff
@@ -81,6 +98,9 @@ oui-table is a package which provides styles for the table component.
       <td data-title="Id" class="oui-table__cell">3</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 7]
```diff
@@ -89,6 +109,9 @@ oui-table is a package which provides styles for the table component.
       <td data-title="Id" class="oui-table__cell">4</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 8]
```diff
@@ -97,12 +120,18 @@ oui-table is a package which provides styles for the table component.
       <td data-title="Id" class="oui-table__cell">5</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md [hunk 9]
```diff
@@ -97,12 +120,18 @@ oui-table is a package which provides styles for the table component.
...
+      </td>
     </tr>
     <tr class="oui-table__row oui-table__row_closed" tabindex="0">
       <td data-title="Filename" class="oui-table__cell">abc</td>
       <td data-title="Id" class="oui-table__cell">6</td>
       <td data-title="Size" class="oui-table__cell">123 KB</td>
       <td data-title="Creation" class="oui-table__cell">Sun Mar 19 19:53:24 2017</td>
+      <td class="oui-table__cell oui-table__cell_action">
+        <button class="oui-button oui-button_secondary">More...</button>
```
Comment (antleblanc) [inline packages/oui-table/README.md]: Could you please add this following attribut `type="button"`?

File: packages/oui-table/README.md (hunk 1 above)
Comment (neolitec) [inline packages/oui-table/README.md]: done (x9)

**Pull request merged**